### PR TITLE
Dock to emit signal when closed

### PR DESCRIPTION
**Pull request by lidstrom83**

### CODE REVIEW

File: pyqtgraph/dockarea/DockArea.py [hunk 1]
```diff
@@ -317,5 +321,12 @@ def dragLeaveEvent(self, *args):
     def dropEvent(self, *args):
         DockDrop.dropEvent(self, *args)
 
-        
-        
+
+class TempAreaWindow(QtGui.QMainWindow):
+    def __init__(self, area, **kwargs):
+        super().__init__(**kwargs)
```
Comment (campagnola) [inline pyqtgraph/dockarea/DockArea.py]: `super()` without arguments is not allowed in python 2.

**Comment by lidstrom83** [comment]
Whoops, shouldn't have been using super() in the first place with PyQt4/PySide...  Fixed now.

**Comment by campagnola** [comment]
Another down. I am digging all these compact PRs  :)

**Comment by lidstrom83** [comment]
Oh good, I was hoping to make it easy on you since I knew I'd have a bunch.

**Comment by campagnola** [comment]
It worked. If those three PRs had all been joined into one, I probably would have taken a lot longer to review them (and they probably would have been more difficult to review anyway).

**Pull request merged**